### PR TITLE
Added a way to check if scanner supports scanButtonRelease. 

### PR DESCRIPTION
**Pull request by Chrishon-Socket**
If the device does not support scanButtonRelease, a 5 second timer will stop the scanning animation. This is a temporary fix until a none enum value is added.

### CODE REVIEW

**Comment by Chrishon-Socket** [comment]
closes #9

File: Over21/ViewController.swift [hunk 1]
```diff
@@ -67,42 +73,57 @@ class ViewController: UIViewController {
 }
 
 extension ViewController: CaptureHelperDelegate {
-
+    
 }
 
 extension ViewController: CaptureHelperDeviceManagerPresenceDelegate {
```
Comment (EricGlaenzer) [inline Over21/ViewController.swift]: If we don't plan to use the D600 in this application, we can remove this delegate.

Comment (Chrishon-Socket) [reply Over21/ViewController.swift]: Removed in latest commit.

Comment (EricGlaenzer) [reply Over21/ViewController.swift]: Did you push it because I don't see it?

Comment (Chrishon-Socket) [reply Over21/ViewController.swift]: Yep, pushed 15 minutes ago.

**Comment by EricGlaenzer** [comment]
This branch has a conflict, can you check this out?

File: Over21/ViewController.swift (hunk 1 above)
Comment (EricGlaenzer) [reply Over21/ViewController.swift]: I think it's still there (the device manager notifications)...

**Pull request merged**